package-lock.json

```diff
@@ -4716,7 +4715,7 @@
         "glslify": "^6.1.0",
         "robust-orientation": "^1.1.3",
         "split-polygon": "^1.0.0",
-        "vectorize-text": "^3.2.0"
+        "vectorize-text": "git://github.com/archmoj/vectorize-text.git#805784cb822eb29096555be6d047b45370650e9e"
```

Comment:
@etpinard what do you think if I may open a PR to merge this one to Mikola's repo now?
https://github.com/mikolalysenko/vectorize-text/compare/master...archmoj:3379-vectorize-text-functions